Dockerfile.prod

```diff
@@ -0,0 +1,12 @@
+# Dockerfile focused on production use case
+# Builder stage needs JDK and gradle
+FROM openjdk:8-jdk-alpine as builder
+WORKDIR /root
+COPY . .
+RUN ./gradlew build
```

Comment:
Until we put this to use and mount the host's Gradle cache, we'll download and build everything from scratch with every build via this Dockerfile